config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
@bbuckingham Thanks for testing it. Does it mean we want to keep puppet: true in by default?